spark/spark-tensorflow-connector/pom.xml

```diff
@@ -4,9 +4,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.tensorflow</groupId>
-    <artifactId>spark-tensorflow-connector_${scala.binary.version}</artifactId>
+    <artifactId>spark-tensorflow-connector_2.12</artifactId>
```

Comment:
Can this continue to use ${scala.binary.version}?

Reply:
Yes, it appears to work. There is some confusion around it though: https://github.com/davidB/scala-maven-plugin/wiki/Frequently-Asked-Questions#cross-building-with-maven

When I try locally, with the variable interpolation of `spark-tensorflow-connector_${scala.binary.version}`, I'm able to run `mvn install` but it issues a warning:

```
[INFO] Scanning for projects...
[WARNING]
[WARNING] Some problems were encountered while building the effective model for org.tensorflow:spark-tensorflow-connector_2.12:jar:1.15.0
[WARNING] 'artifactId' contains an expression but should be a constant. @ org.tensorflow:spark-tensorflow-connector_${scala.binary.version}:1.15.0, /Users/groodt/dev/ecosystem/spark/spark-tensorflow-connector/pom.xml, line 7, column 17
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
```

I've looked at the `pom.xml` in my `~/.m2` folder and it looks ok, so I think this warning is fine. This is with Maven 3.6.3.

If you'd prefer that I keep the interpolation, let me know and I'll change it. There doesn't seem to be great support in Maven for doing cross-builds for Scala.

I hardcoded it since Spark 3.x is only Scala 2.12 at the moment. It does not support 2.11 or 2.13.